Module-generation/frontmatter.py

```diff
@@ -0,0 +1,75 @@
+import os
+import json
+
+DIR = "./meta-data/"
+DST = "./modules/"
+os.mkdir(DST)
+
+for folder in os.listdir(DIR):
+    moduleFile = DIR+folder+"/module.txt"
+    # check for Module.txt to get infomation about module
+    if(os.path.isfile(moduleFile)):
+        getModuledata = open(moduleFile, "r")
+        readModuleData = getModuledata.read()
+        parseData = json.loads(readModuleData)
+        moduleName = parseData['id']
+        moduleDescription = parseData['description']
+        moduleLogo = "defaultBanner.png"
```

Comment:
Why do you statically use the fallback banner here?
For one, you're currently handling the case in which no banner is present already on scrape and copy the default banner, right? And secondly, even if you didn't, shouldn't you first check whether there is a banner present in the scrape result?

Reply:
fixed